modules/apps/digital-signature/digital-signature-impl/src/main/java/com/liferay/digital/signature/internal/manager/DSEnvelopeManagerImpl.java

```diff
@@ -127,6 +127,37 @@ public DSEnvelope getDSEnvelope(long groupId, String dsEnvelopeId) {
 			evenlopeJSONObject -> _toDSEnvelope(evenlopeJSONObject), _log);
 	}
 
+	@Override
+	public JSONObject getDSEnvelopesJSONObject(
+		long groupId, String fromDateString, long count, long startPosition) {
+
+		JSONObject jsonObject = _dsHttp.get(
+			groupId,
+			StringBundler.concat(
+				"envelopes?from_date=", fromDateString, "&count=",
+				String.valueOf(count), "&start_position=",
+				String.valueOf(startPosition),
+				"&include=custom_fields,documents,recipients&order=desc"));
```

Comment:
This `order=desc` could be a param received from Request.

An optional param `sort`, we could use `desc` or `asc` as default.